pyproject.toml

```diff
@@ -0,0 +1,49 @@
+
+[project]
+name = "safeeyes"
+version = "2.2.2"
+description = "Protect your eyes from eye strain using this continuous breaks reminder."
+keywords = ["linux utility health eye-strain safe-eyes"]
+readme = "README.md"
+authors = [
+    {name = "Gobinath Loganathan", email = "slgobinath@gmail.com"},
+]
+classifiers = [
```

Comment:
Task for a different PR - enrich `project` section with useful information using references like [fastapi](https://pypi.org/project/fastapi/) and other popular tools